src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -23,8 +23,10 @@ class Lambda {
       (event.itemType, event.eventType) match {
         case (ItemType.Content, EventType.Delete) =>
           sendPurgeRequest(event.payloadId)
+        case (ItemType.Content, EventType.Update) =>
+          sendSoftPurgeRequest(event.payloadId)
         case other =>
-          // for now we only send purges for content takedowns, so ignore any other events
+          // for now we only send purges for content, so ignore any other events
```

Comment:
@johnduffell @gklopper Out of interest, are there additional things that we could help you purge that we can't at current. I've always planned to enrich the types of events sent onto the firehose so I'm happy to take requests.